python/samples/learn_resources/__init__.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Microsoft. All rights reserved.
 
-from .service_settings import ServiceSettings
-from .sk_service_configurator import add_service
+from ..service_settings import ServiceSettings
```

Comment:
How come the previous one wasn't detected as incorrect by the CI? Could we have a method to check all the imports are valid on the CI?

Reply:
No I moved the files for the settings, and the purpose of this is to start automatically checking the samples!